### ISSUE TITLE
Well-known URI for password changes

### ISSUE THREAD

**Issue by jkcclemens** [jkcclemens/paste#105]
See [here](https://github.com/WICG/change-password-url/blob/gh-pages/explainer.md). Should paste implement this? I see no harm in it.

**Comment by AKPWebDesign** [comment]
Yeah, same. I'm sure it's simple enough to 301 redirect `/.well-known/change-password` to the password change page, and I can't think of anything that would be hurt by doing it.
